presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -1951,6 +1951,20 @@ public void testScaleWriters()
         }
     }
 
+    @Test
+    public void testTableCommentsTable()
+    {
+        assertUpdate("CREATE TABLE test_comment (c1 bigint) COMMENT 'foo'");
+        String selectTableComment = format("" +
+                                           "SELECT comment FROM system.metadata.table_comments " +
```

Comment:
Nit: this indentation is off (should be an 8 space continuation indent, not aligned). IntelliJ formatter should fix this.